app/lib/matador/components/job-data/JobData.tsx

```diff
@@ -0,0 +1,34 @@
+import { Card, Grid, JsonInput, Text } from "@mantine/core";
+
+interface JobDataProps {
```

Comment:
export the type

Reply:
Fixed with 35c6421357701641e9eedeec5819dece439e234c